src/context.ts

```diff
@@ -388,6 +388,44 @@ export class Context implements RenamedUpdate {
                 this.chosenInlineResult?.inline_message_id
         );
     }
+    /**
+     * Get entities and their text. Extracts the text from `ctx.msg.text` or `ctx.msg.caption`.
+     * Returns an empty array if one of `ctx.msg`, `ctx.msg.text`
+     * or `ctx.msg.entities` is undefined.
```

Comment:
Hmm... what do you think about putting it this way?

```suggestion
     * Returns an empty array if `ctx.msg` or `ctx.msg.entities` is undefined
     * or if `ctx.msg.text` and `ctx.msg.caption` are both undefined.
```

Reply:
what about using the `?.` operator to simplify the docs: `ctx.msg?.entities`?

Reply:
How would that look?

Reply:
Maybe like this?

```suggestion
     * Returns an empty array when `ctx.msg` is undefined or no entities are within it.
```

Thinking about it for the documentations its only relevant to know that it falls back to an empty array rather than undefined. Just talking about entities after the first sentence means both `entities` and `caption_entities`. I think this is a pretty simple to understand sentence that explains what is needed to know?